src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>I think you mean the pudl database tables are a standard <strong>normalized</strong> way to store and access electricity data. For the non RDBMS geeks, explaining (or offering a link to an additional resource) about what normalized vs. denormalized tables are would be helpful I think. Mostly it's including the names of referenced utilities and plants, instead of just their IDs, and then also some frequently calculated columns (and grab a couple of examples...)</p><p> </p><p>We've associated pudl@catalyst.coop with the maintainers of the project (which might not just be us someday) rather than our generic hello@catalyst.coop email. I think we probably want feedback / info here to go to pudl@catalyst.coop</p><p> </p><p>Only annual and monthly aggregation works right now, so we probably want to make that very clear -- since there are lots of other frequencies available in the generic pandas time series analysis toolbox.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>